src/map/packets_struct.h

```diff
@@ -3748,17 +3748,44 @@ struct PACKET_ZC_NOTIFY_EFFECT3 {
 DEFINE_PACKET_HEADER(ZC_NOTIFY_EFFECT3, 0x0284);
 #endif
 
+#if PACKETVER >= 20100824
+struct PACKET_CZ_SE_CASHSHOP_OPEN1 {
+	int16 packetType;
```

Comment:
Actually I'd ask for lowercase names again, but it seems like `packetType` is spelled like this in the whole file.
Is there a reason to do so, or is it just because it was always done like this.

Reply:
isn't is better to seperate two different words either by camelCase or adding underscore?